sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/src/Microsoft.Azure.WebJobs.Extensions.SignalRService.csproj

```diff
@@ -11,24 +11,24 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <!--Pin the versions of packages that may be upgraded during the process of Azure SDK moving all dependencies to the 8.x line to avoid incompatibility, until a compatible dependency gragh is figured out and finalized.-->
     <PackageReference Include="MessagePack" />
-    <PackageReference Include="Microsoft.Extensions.Azure" />
-    <PackageReference Include="Microsoft.Azure.Functions.Extensions" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" VersionOverride="3.0.37"/>
     <PackageReference Include="Microsoft.Azure.SignalR.Management" />
     <PackageReference Include="Microsoft.Azure.SignalR" />
     <PackageReference Include="Microsoft.Azure.SignalR.Protocols" />
     <PackageReference Include="Microsoft.Azure.SignalR.Serverless.Protocols" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" />
+    <PackageReference Include="Microsoft.Extensions.Azure" VersionOverride="1.10.0"/>
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" VersionOverride="6.35.0"/>
```

Comment:
This override is not necessary.  It matches the [central version](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Packages.Data.props#L178).    Please remove.

Reply:
For these overrides that match the central version, I want to use them to avoid unexpected central version upgrade that breaks the current build in the short term. Will remove after this release.

Reply:
As stated below, you don't want v1.10.0 of the extensions package, then.

Reply:
Continued the discussion here: https://github.com/Azure/azure-sdk-for-net/pull/48280#discussion_r1969251639